### ISSUE TITLE
Why are the scss files in the prime-designer/assets/designer folder not a part of the PrimeNG project

### ISSUE THREAD

**Issue by kervi** [primefaces/primeng#7377]
Why are the scss files in the prime-designer/assets/designer folder not a part of the PrimeNG project. As these files are not open source it is very difficult to create PrimeNG themes and distribute them with open source projects.

I am working on an open source project https://github.com/kervi/kervi and have started to implement PrimeNG in the UI part, the main reason is that PrimeNG offer all the components that is needed in my project . Secondly I thought that I could create a scss theme and distribute it with my project allowing users of my project to tweek it to own preferences.

Now I am in doubt if I shall continue to use PrimeNG. I known that I can distribute the compiled css files, and my theme scss files with my project. But some how it doesn't feels like open source because of the dependency on a closed source product that should be payed for.

I do understand that you have an revenue stream from creating good looking themes but I don't think that you will undercut your own business by open source the core theme api scss files. It takes long time and great skills to create the themes that you sell so it is a lot cheaper to just buy your themes than try to build a theme my self.

Also I have no problem with buying the designer as it speeds up the work on creating my own theme for my open source project.

**Comment by Dorwido** [comment]
Like alot of open source, first have all included (from log the theme files were scss), when you have enough user, then make either an pro version or like in this case, even remove things that was there before.

Personally the time were I edited css at dozen of places just to change colors are over and thats why I decided against this.

Found then this:
https://github.com/NG-ZORRO/ng-zorro-antd

Its under MIT license, less files included and alot of components, quite similar to this, without the restrictions of this, maybe that fits you as well.
